lms/djangoapps/courseware/tests/test_course_info.py

```diff
@@ -367,7 +367,7 @@ def fetch_course_info_with_queries(self, course, sql_queries, mongo_queries):
         self.assertEqual(resp.status_code, 200)
 
     def test_num_queries_instructor_paced(self):
-        self.fetch_course_info_with_queries(self.instructor_paced_course, 23, 4)
+        self.fetch_course_info_with_queries(self.instructor_paced_course, 24, 4)
```

Comment:
Is this increased query count from `social_auth_partial` ? If so, any ideas why we're even checking that for something like this, where we're not actively trying to complete a third_party_auth login?

I don't really like the fact that this upgrade seems to be increasing query counts in unrelated areas of the system, though I know it's likely not a big performance concern.